podaac/swagger/swagger.yaml

```diff
@@ -0,0 +1,225 @@
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+          enum: [ "Reach", "Lake", "Node"]
+        example: Reach
+      - name: subsetpolygon
+        in: query
+        description: GEOJSON of the subset area
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+        example: '{"features": [{"type": "Feature","geometry": {"coordinates": [[-99.85255374863124,41.39088502467528],[-98.05255374863124,41.39088502467528],[-98.05255374863124,38.99088502467528],[-99.85255374863124,38.99088502467528],[-99.85255374863124,41.39088502467528]],"type": "LineString"},"properties": {}}],"type": "FeatureCollection"}'
+      - name: startTime
+        in: query
+        description: Start time of the timeseries
+        required: true
+        style: form
+        explode: true
+        schema:
+          type: string
+          format: date-time
+        example: 2022-08-04T00:00:00Z
+      - name: endTime
+        in: query
+        description: End time of the timeseries
+        required: true
+        style: form
+        explode: true
+        schema:
+          type: string
+          format: date-time
+        example: 2022-08-23T00:00:00Z
+      - name: output
+        in: query
+        description: Format of the data returned
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+          enum: [ "csv", "geojson"]
+          default: geojson
+        example: geojson
+      - name: fields
+        in: query
+        description: Format of the data returned
+        required: false
+        style: form
+        explode: true
+        schema:
+          type: string
+          default: feature_id, time_str, wse, geometry
+        example: feature_id, time_str, wse, geometry
+      responses:
+        "200":
+          description: OK
+          content:
+            text/csv:
+              schema:
+                type: array
+                items:
+                  type: string
+        "400":
+          description: "400 error. The specified URL is invalid (does not exist)."
+          content:
+            text/csv:
+              schema:
+                type: array
+                items:
+                  type: string
+        "404":
+          description: "404 error. An entry with the specified region was not found."
+          content:
+            text/csv:
+              schema:
+                type: array
+                items:
+                  type: string
+        "413":
+          description: "413 error. Your query has returned is too large."
+          content:
+            text/csv:
+              schema:
+                type: array
+                items:
+                  type: string
+      x-openapi-router-controller: podaac.controllers.default_controller
+components:
+  schemas: {}
+  securitySchemes:
+    application:
+      type: oauth2
```

Comment:
We haven't discussed authentication yet; I think we should remove this from the spec